agogosml/Makefile

```diff
@@ -52,6 +52,7 @@ clean-test: ## remove test and coverage artifacts
 
 lint: ## check python style
 	flake8 agogosml tests
+	pylint agogosml
```

Comment:
Will Pylint run against the tests/ directory as well?

Also, is flake8 still needed? Does pylint cover the same rules as flake8?

Reply:
There's some overlap but they check for different things. Right now it's not running against tests.

Reply:
The tests have *many* pylint errors. Fixing them all right now would be pretty expensive.

Reply:
👍